package-lock.json

```diff
@@ -197,6 +198,199 @@
 				"webpack-rtl-plugin": "2.0.0"
 			},
 			"dependencies": {
+				"@babel/compat-data": {
```

Comment:
We'll need to make these package-lock updates eventually, but let's let https://github.com/Automattic/wp-calypso/pull/39420 handle that.

Can you rollback these package lock changes?